app/src/main/java/com/medmapper/v33001/dto/Medicine.kt

```diff
@@ -9,7 +9,7 @@ data class Medicine(
     @PrimaryKey(autoGenerate = true) val medID: Int,
     @ColumnInfo(name = "name") val name: String?,
     @ColumnInfo(name = "strength") val strength: String?,
-    @ColumnInfo(name = "start date") val startDate: String?,
-    @ColumnInfo(name = "prescription length") val lengthInDays: Int
+    @ColumnInfo(name = "start_date") val startDate: String?,
+    @ColumnInfo(name = "prescription_length") val lengthInDays: Int
```

Comment:
It's important to seperate the column names with a "_" inbetween.

source: https://developer.android.com/training/data-storage/room/defining-data